DQM/SiStripCommissioningSources/plugins/tracking/SiStripFineDelayHit.cc

```diff
@@ -119,51 +114,53 @@ SiStripFineDelayHit::~SiStripFineDelayHit()
 //
 // member functions
 //
-std::pair<uint32_t, uint32_t> SiStripFineDelayHit::deviceMask(const StripSubdetector::SubDetector subdet,const int substructure)
+std::pair<uint32_t, uint32_t> SiStripFineDelayHit::deviceMask(const StripSubdetector::SubDetector subdet,const int substructure,const TrackerTopology* tkrTopo)
 {
   uint32_t rootDetId = 0;
   uint32_t maskDetId = 0;
+
+
   switch(subdet){
     case (int)StripSubdetector::TIB :
```

Comment:
while we're at it...  how about removing the cast to int? There's an unsigned 32-bit wide integer on both sides (small enough that a cast to signed should not do any harm, but still not necessary, I think)